gnomad_qc/v3/resources/release.py

```diff
@@ -58,9 +58,9 @@ def release_ht_path(
     :rtype: str
     """
     if public:
-        return f"gs://gnomad-public/release/{release_version}/ht/{data_type}/gnomad.{data_type}.r{release_version}.sites.ht"
+        return f"gs://gnomad-public/release/{release_version}/ht/{data_type}/gnomad.{data_type}.v{release_version}.sites.ht"
     else:
-        return f"gs://gnomad/release/{release_version}/ht/gnomad.{data_type}.r{release_version}.sites.ht"
+        return f"gs://gnomad/release/{release_version}/ht/{data_type}/gnomad.{data_type}.v{release_version}.sites.ht"
```

Comment:
I think this currently doesn't work for any of the 3 releases (3.1 and 3.1.1 don't have hts in this bucket, and 3.0 has an 'r' instead of a 'v') -- not sure if this matters right now and will be fixed later